Makefile

```diff
@@ -39,7 +39,7 @@ push:
 	@$(CONTAINER_ENGINE) --config=$(DOCKER_CONF) push $(IMAGE_NAME):$(IMAGE_TAG)
 
 rc: git_version
-	@$(CONTAINER_ENGINE) build -t $(IMAGE_NAME):$(IMAGE_TAG)-rc -f dockerfiles/Dockerfile --target prod-image .
+	@$(CONTAINER_ENGINE) build -t $(IMAGE_NAME):$(IMAGE_TAG)-rc --build-arg quay_expiration=3d -f dockerfiles/Dockerfile --target prod-image .
```

Comment:
`rc` looks like a temp tag, when are we using `make rc`?

Reply:
rc image can be used for testing in app-interface pr-check, especially when the code change may take effect on multiple integrations.